### PR TITLE
Tickets/sbir custom homepage

### PR DESCRIPTION
**Pull request by erikscottmeyer**
Two tickets for the SBIR Custom homepage

### CODE REVIEW

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_borderless-cards.scss [hunk 1]
```diff
@@ -134,7 +134,7 @@ a.borderless-button {
 
 //accent colors
 .accent-cool {
-  color: $white;
+  //color: $white;
```
Comment (arcepaul) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_borderless-cards.scss:137]: just remove

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_borderless-cards.scss [hunk 2]
```diff
@@ -149,11 +149,34 @@ a.borderless-button {
     h2 >a {
       color: $white;
     }
+    h1 > a,
+    h2 > a {
+      &:hover {
+        color: $white;
+        //color: $verydarkblue;
```
Comment (arcepaul) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_borderless-cards.scss:156]: remove

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_sbir.scss [hunk 3]
```diff
@@ -0,0 +1,82 @@
+@import "./variables";
+@import "./mixins";
+@import "./environments/percussion";
+@import "./breakpoints";
+
```
Comment (arcepaul) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_sbir.scss:5]: It feels like moving to a BEM naming convention would help organization and future maintenance here

**Comment by alinai** [comment]
@blairlearn , ODE 458 has a content difference on https://ncigovcdode458.prod.acquia-sites.com/news-events page w.r.t dev-ac (https://www-dev-ac.cancer.gov/news-events).

ODE url: https://ncigovcdode458.prod.acquia-sites.com/news-events
dev-ac url:  https://www-dev-ac.cancer.gov/news-events

image

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/ckeditor/Ckeditor.scss [hunk 4]
```diff
@@ -98,6 +98,7 @@ CSS modules.
 @import "~Core/styles/checkbox";
 @import "~Core/styles/popups";
 @import '~Core/styles/tableSort';
+@import '~Core/styles/sbir';
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/ckeditor/Ckeditor.scss:101]: Can you remove the reference to SBIR in commit 1f927bdd27c8de99f539c9f523bb4670859c77d1. This is not part of ticket 3311.

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/global/Common.scss [hunk 5]
```diff
@@ -91,6 +91,7 @@
 @import '~Core/styles/alternating-lists';
 @import '~Core/styles/special-report-raw';
 @import '~Core/styles/title-first-feature-cards';
+@import '~Core/styles/sbir';
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/global/Common.scss:94]: Can you remove the reference to SBIR in commit 1f927bdd27c8de99f539c9f523bb4670859c77d1. This is not part of ticket 3311.

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]
I found a couple of things listed below. 

Also, I don't think we need to make a SBIR stylesheet for these 2 raw HTML blocks. You can add a `<style>` to the raw HTML. It most certainly should not be in common.css if it does stay. This would only need to be referenced in homelanding.css if it *needs* to be a css file. (Common is referenced on every page of every website we have)

**Comment by arcepaul** [comment]
We're missing alt text on the homepage images which is showing up as errors on accessibility scan.

**Comment by arcepaul** [comment]
@acsnedegar is there an svg version of event-placeholder-icon?  It doesn't look as crisp as it should, right?

**Comment by acsnedegar** [comment]
@arcepaul - these icons will need to be updated with final icons from SBIR so I wouldn't worry about swapping in an svg now. We can be sure to ask for an svg from them for this block though. Thanks!

**Comment by bryanpizzillo** [comment]
> @arcepaul - these icons will need to be updated with final icons from SBIR so I wouldn't worry about swapping in an svg now. We can be sure to ask for an svg from them for this block though. Thanks!

@acsnedegar If the image is being committed to this code base is *must* be final.

**Comment by bennettcc** [comment]
The PR has been updated to add these icons (svg format) inline. This has been retested and passed on the ODE 446

**Comment by arcepaul** [comment]
@erikscottmeyer , could you please squash those commits for #3314?  this PR should only have 2 commits, one for #3324 and one for #3311

**Comment by erikscottmeyer** [comment]
This has been replaced by PR 3347: https://github.com/NCIOCPL/cgov-digital-platform/pull/3347 (needed to squash commits to one per ticket)

**Pull request closed without merging**